src/cart/SConscript

```diff
@@ -64,6 +64,8 @@ def scons():
     env.AppendUnique(LIBPATH=[Dir('.')])
 
     env.require('mercury', headers_only=True)
+    env.require('isal', headers_only=True)
+    env.require('argobots', headers_only=True)
```

Comment:
why is this needed?   I recently pushed a PR to make sure argobots is only needed for client build

Reply:
one of these headers was pulling dependencies -- i ll need to go back check which one was it
#include "../mgmt/rpc.h"
#include "../pool/rpc.h"
#include "../container/rpc.h"
#include "../pipeline/pipeline_rpc.h"
#include "../object/obj_rpc.h"

Reply:
argobots was being pulled by rdb header file, however it starts wanting to pull in flake and other things too, and for that reason for now the rdb module is not included in rpc prints. 
isal was required by object/obj_rpc.h which then includes object/obj_ec.h 

i've removed argobots for now, but have to keep isal for object module